src/main/java/technology/tabula/CommandLineApp.java

```diff
@@ -290,7 +285,15 @@ private static TableExtractor createExtractor(CommandLine line) throws ParseExce
         List<Float> rv = new ArrayList<>();
         try {
             for (int i = 0; i < f.length; i++) {
-                rv.add(Float.parseFloat(f[i]));
+                final String element = f[i];
+
+                if (element.startsWith("%")) {
+
+                    rv.add(Float.parseFloat(element));
```

Comment:
Why both branches execute the same code?

Reply:
You are right. Sorry, I didn't want left this code behind in that way. It was by accident.